src/tools/miri/src/helpers.rs

```diff
@@ -584,8 +583,9 @@ pub trait EvalContextExt<'tcx>: crate::MiriInterpCxExt<'tcx> {
             fn visit_value(&mut self, v: &MPlaceTy<'tcx>) -> InterpResult<'tcx> {
                 trace!("UnsafeCellVisitor: {:?} {:?}", *v, v.layout.ty);
                 let is_unsafe_cell = match v.layout.ty.kind() {
-                    ty::Adt(adt, _) =>
-                        Some(adt.did()) == self.ecx.tcx.lang_items().unsafe_cell_type(),
+                    ty::Adt(adt, _) => {
+                        Some(adt.did()) == self.ecx.tcx.lang_items().unsafe_cell_type()
+                    }
```

Comment:
Here and below please revert the reformatting. Miri has different formatting rules.